app/views/courses/CoursesView.coffee

```diff
@@ -241,3 +245,10 @@ module.exports = class CoursesView extends RootView
     courseID = $(e.target).data('course-id')
     window.tracker?.trackEvent 'Students View To Student Assessments View', category: 'Students', classroomID: classroomID, ['Mixpanel']
     application.router.navigate("/students/assessments/#{classroomID}##{courseID}", { trigger: true })
+  
+  onClickViewVideosLink: (e) ->
+    classroomID = $(e.target).data('classroom-id')
```

Comment:
What do we think about storing attributes on the DOM element like this?

My minor concern/thoughts are that it maybe introduces an implicit dependency.
It's not explicit to trace the anchor tag + its attributes to the routing.

This appears to be the way it's been done so I think it's fine. I wonder if there is an alternative way.